### PR TITLE
Fixes Register Items exception for missing method object_location

### LINKED ISSUE #320: Register Items page has broken javascript
Reported originally by @drh-stanford here: https://github.com/sul-dlss/argo/pull/315#issuecomment-163004597

JavaScript functionality in the items registration page is broken, thus causing additional JavaScript to run.

### PR DESCRIPTION
**Pull request by drh-stanford**
This PR fixes the Register Items exceptions, does some cleanup, and adds a development fixture for the `digitizationWF`. The cleanup is that we were only implementing the create (POST) to `/dor/objects` that's used by the Register Items UI, so I removed the unimplemented routes. I also moved `object_location` from ApplicationHelper to a private method inside the controller. There was also no spec file corresponding to the objects controller, so I created a stub spec.

Fixes #320

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -162,11 +162,14 @@
     match 'reindex/:pid',   :action => :reindex, :as => 'reindex', :via => [:get, :post]
     match 'delete_from_index/:pid', :action => :delete_from_index, :via => [:get, :post]
     get 'index_exceptions'
-    resources :objects
+    namespace :objects do
```
Comment (mejackreed) [inline config/routes.rb]: If we are implementing create only, maybe:
`resources :objects, only: :create`
?

That way it keeps the resourceful structure without just tacking it on a namespace?

File: spec/controllers/dor/objects_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+require 'spec_helper'
```
Comment (mejackreed) [inline spec/controllers/dor/objects_controller_spec.rb]: Is this an autogen file? If we aren't testing anything do we need this?

**Comment by drh-stanford** [comment]
I tested this on -dev and it worked as expected. It's a bit difficult to test but I did add the `digitizationWF` fixtures so I was able to test it on my laptop.

**Comment by drh-stanford** [comment]
The coverage went down but only on files I didn't touch?

**Comment by mejackreed** [comment]
Strange on the coverage. Now that I look at this again, I'm wondering if a quick controller spec testing `create` would have caught that helper include regression. Is it possible to add in?

**Comment by drh-stanford** [comment]
this should be ready to go...

**Comment by mejackreed** [comment]
:+1: :shipit:

**Pull request merged**